app/concepts/movies/use_cases/show_all.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Movies
+  module UseCases
+    class ShowAll
```

Comment:
This is the same as with the `ShowIndex` use case. You can use the representer method.